utils.lua

```diff
@@ -150,4 +150,16 @@ function nn.utils.addSingletonDimension(t, dim)
   return view
 end
 
+function nn.utils.contiguousView(output, input, ...)
+  output = output or input.new()
+  if input:isContiguous() then
+    output:view(input, ...)
+  else
+    output:resizeAs(input)
+    output:copy(input)
+    output:view(...)
```

Comment:
output:view(...) will enter the following if: 
elseif istensor(result) and type(src) == 'number' then
...
view = tensor.new()
...
view:set(tensor:storage(), tensor:storageOffset(), size)

Why do you think this is not what we need?